lib/webview_screen.dart

```diff
@@ -0,0 +1,102 @@
+import 'dart:convert';
+import 'package:calculator_flutter/bloc/webview/webview_bloc.dart';
+import 'package:calculator_flutter/bloc/webview/webview_event.dart';
+import 'package:calculator_flutter/bloc/webview/webview_state.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class TradingViewWidget extends StatelessWidget {
+  const TradingViewWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => WebViewBloc()..add(LoadWebView()),
+      child: Scaffold(
+        backgroundColor: Colors.white,
+        appBar: AppBar(
+          title: const Text('TradingView Chart'),
+          backgroundColor: Colors.white,
+        ),
+        body: BlocBuilder<WebViewBloc, WebViewState>(
+          builder: (context, state) {
+            if (state is WebViewLoading) {
+              return const Center(child: CircularProgressIndicator());
+            } else if (state is WebViewLoaded) {
+              final controller = WebViewController()
+                ..setJavaScriptMode(JavaScriptMode.unrestricted)
+                ..setNavigationDelegate(
+                  NavigationDelegate(
+                    onProgress: (int progress) {},
+                    onPageStarted: (String url) {},
+                    onPageFinished: (String url) {},
+                    onHttpError: (HttpResponseError error) {},
+                    onWebResourceError: (WebResourceError error) {},
+                  ),
+                )
+                ..loadRequest(
+                  Uri.dataFromString(
```

Comment:
Better to move it in separate json file and load the file here instead of plain text.